src/server/components/dashboard/controllers/listsController.helpers.ts

```diff
@@ -1,57 +1,46 @@
-import type { ListsForDashboard, Prisma } from "@prisma/client";
-import pluralize from "pluralize";
-import type { NextFunction, Request, Response } from "express";
-import { authRoutes } from "server/components/auth";
-import { pageTitles } from "server/components/dashboard/helpers";
-import { dashboardRoutes } from "server/components/dashboard/routes";
-import { getCSRFToken } from "server/components/cookies/helpers";
+import { Request } from "express";
 import Joi from "joi";
-import { countriesList } from "server/services/metadata";
-import { ServiceType } from "server/models/types";
+import { ListsForDashboard, Prisma } from "@prisma/client";
+import pluralize from "pluralize";
 
 type DashboardOrderByInput = Omit<Prisma.ListsForDashboardOrderByWithRelationInput, "listId" | "jsonData">;
-export const DEFAULT_VIEW_PROPS = {
-  dashboardRoutes,
-  countriesList,
-  ServiceType,
-};
-export async function listsController(req: Request, res: Response, next: NextFunction) {
-  try {
-    if (req.isUnauthenticated()) {
-      res.redirect(authRoutes.logout);
-      return;
-    }
-
-    const orderBy = calculateSortOrder(req.query);
-
-    const lists = (await req.user?.getLists(orderBy)) ?? [];
-    const isNewUser = !req.user?.isAdministrator && lists?.length === 0;
```

Comment:
length check is redundant now. user.getLists will always come back with >1 result if you are an admin.